packages/@react-types/card/src/index.d.ts

```diff
@@ -26,6 +26,21 @@ interface SpectrumCardProps extends AriaCardProps, StyleProps, DOMProps {
   orientation?: Orientation
 }
 
+export interface CardViewCardProps {
+  /** Rendered contents of the card. Note that focusable elements are not allowed within the card in a CardView. */
+  children: ReactNode,
+  /** A string representation of the cards's contents, used for features like typeahead. */
+  textValue?: string,
+  /** An accessibility label for this card. */
+  'aria-label'?: string,
+  /** The raw height of the card's image. */
+  height?: number,
+  /** The raw width of the card's image. */
+  width?: number,
+  /** A unique key for the card. */
+  key?: Key
+}
```

Comment:
A more contained list of Card props that are valid for a Card within a CardView. Would be easier if we split the standalone Card into a separate component